rspace/src/main/scala/coop/rchain/rspace/merger/instances/DiffStateMerger.scala

```diff
@@ -0,0 +1,317 @@
+package coop.rchain.rspace.merger.instances
```

Comment:
Sorry for gigantic file, but there are 4 parts here - 
1. preparation of Refs to throw workers on them
2. preparation of worker streams
3. populating Refs
4. Calculating storage actions from Refs 
Maybe split this later when we have more abstract merging API.